public/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Calculator Application</title>
 <link rel="stylesheet" type="text/css" href="style.css">
+<script src="//cdn.bootcss.com/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
With no longer using jquery, should remove the link to it

Reply:
ok